Client/Frontend/Home/FirefoxHomeTopSitesCell.swift

```diff
@@ -413,7 +413,7 @@ class ASHorizontalScrollCellManager: NSObject, UICollectionViewDelegate, UIColle
             return 0
         }
         let isLandscape = UIApplication.shared.statusBarOrientation.isLandscape
-        if UIDevice.current.userInterfaceIdiom == .phone {
+        if UIDevice.current.isPhone {
```

Comment:
I suggest replacing this with a UITraitCollection based approach. You could be on an iPad, but in split screen mode.